src/vs/workbench/workbench.common.main.ts

```diff
@@ -394,6 +394,9 @@ import 'sql/workbench/contrib/query/common/resultsGrid.contribution';
 import 'sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution';
 import 'sql/workbench/contrib/dataExplorer/browser/nodeActions.common.contribution';
 
+// notebook explorer
+import 'sql/workbench/contrib/notebooksExplorer/browser/notebookExplorer.contribution';
```

Comment:
I would just reuse the existing notebooks contribution file. at the very least this import should be next to the other notebook contribution import.